docs/core/development/policy/release-process.rst

```diff
@@ -107,9 +107,9 @@ How to do a pre-release
 
 1. Check ​buildbot to make sure all supported platforms are green (wait for pending builds if necessary).
 2. If a previously supported platform does not currently have a buildbot, move from supported platforms to "expected to work" in ``INSTALL.rst``.
-3. In your Git-SVN-enabled Git repo, fetch and check out the new release branch.
-4. Run ``./bin/admin/change-versions --prerelease``
-5. Commit the changes made by change-versions
+3. In your Git repo, fetch and check out the new release branch.
+4. Run ``python -m incremental.update Twisted --newversion $RELEASErc1``
```

Comment:
The change from "prerelease" to "release candidate" isn't a big deal because only `pip` (and `dpkg` and `pkg` and...) care, right?